hugo/layouts/index.html

```diff
@@ -27,9 +27,7 @@
         {{ partial "home/blog.html" . }}
     {{ end }}
 
-    {{ partial "footer/slack.html" . }}
-    {{ partial "footer/contact-us.html" . }}
-    {{ partial "footer/sitemap.html" . }}
+    {{ partial "footer.html" . }}
```

Comment:
From the https://github.com/src-d/minutes/blob/master/platform/2017-06-06-sprint-10-planning-part-1.md
Sidemap, slack and email are used in different places and only the sidemap will appear in the whole site.

I'd preffer the parametric one if the slack and mail is included too (because all of them are part of the same visual block, and it's possible that they're used in different places)